src/app/components/Metadata/index.test.tsx

```diff
@@ -128,6 +132,19 @@ const CanonicalNewsInternationalOrigin = (
   />
 );
 
+const FacebookDomainVerification = (props: FacebookDomainVerificationProps) => (
+  <MetadataWithContext
+    service="news"
+    bbcOrigin={dotComOrigin}
+    platform="canonical"
+    id="c0000000001o"
+    pageType={ARTICLE_PAGE}
+    pathname="/news/articles/c0000000001o"
+    {...newsArticleMetadataProps}
+    {...props}
+  />
+);
+
```

Comment:
👋 I didn't mean for you to add this - what you had previously was fine. I only meant to rename the variable name in the test... 😬

Reply:
My bad! Sorry I boiled the ocean there!